### DISCUSSION TITLE
Volume mounts (to persist data between container restarts)

### DISCUSSION THREAD

**Discussion by binaryfire** [buggregator/server#262] [Q&A] [answered]
Hi guys

Could you tell me which directory the Buggregator database is in? I'd like to use a volume so data is persisted between container restarts.

Cheers

**Comment by roxblnfk** [comment, marked as answer]
Hi.
Looks like it's `/app/.db`
https://github.com/buggregator/server/blob/76290b296bce6cabb43b85fc26a4cffbe24a03e0/.docker/Dockerfile#L38-L43

**Reply by binaryfire** [reply]
Great thanks!
